packages/jsii/lib/assembler.ts

```diff
@@ -645,18 +645,10 @@ export class Assembler implements Emitter {
   }
 
   /**
-   * Register documentations on a ``spec.Documentable`` entry.
-   *
-   * @param sym       the symbol holding the JSDoc information
```

Comment:
It is sad that you're dropping rich documentation and making it less nice, while you're fixing the problems with it...

Reply:
Didn't feel like it was adding a lot of value. If the parameter is declared as `sym: Symbol` and the documentation basically says "this is the symbol", then I don't feel like the documentation is adding a lot of information.

Similarly for the return value. Declaration says: `function(...): Docs`, I don't feel the need to add `@returns the docs`.